docs/README.md

```diff
@@ -3,101 +3,102 @@ sidebar_position: 1
 sidebar_label: Introduction
 ---
 
-# The Zed Project
+# SuperDB
 
-Zed offers a new approach to data that makes it easier to manipulate and manage
-your data.
-
-With Zed's new [super-structured data model](formats/README.md#2-zed-a-super-structured-pattern),
+SuperDB offers a new approach that makes it easier to manipulate and manage
+your data.  With its [super-structured data model](formats/README.md#2-zed-a-super-structured-pattern),
 messy JSON data can easily be given the fully-typed precision of relational tables
 without giving up JSON's uncanny ability to represent eclectic data.
 
 ## Getting Started
 
-Trying out Zed is easy: just [install](install.md) the command-line tool
-[`zq`](commands/zq.md) and run through the [zq tutorial](tutorials/zq.md).
-
-`zq` is a lot like [`jq`](https://stedolan.github.io/jq/)
-but is built from the ground up as a search and analytics engine based
-on the [Zed data model](formats/zed.md).  Since Zed data is a
-proper superset of JSON, `zq` also works natively with JSON.
+Trying out SuperDB is easy: just [install](install.md) the command-line tool
+[`super`](commands/zq.md) and run through the [tutorial](tutorials/zq.md).
 
-While `zq` and the Zed data formats are production quality, the Zed project's
-[Zed data lake](commands/zed.md) is a bit [earlier in development](commands/zed.md#status).
+Compared to putting JSON data in a relational column, the
+[super-structured data model](formats/zed.md) makes it really easy to
+mash up JSON with your relational tables.  The `super` command is a little
+like [DuckDB](https://duckdb.org/) and a little like
+[`jq`](https://stedolan.github.io/jq/) but super-structured data ties the
+two patterns together with strong typing of dynamic values.
```

Comment:
This was some new content proposed by @mccanne when he responded on Slack after seeing an early draft. Indeed, even as I was doing the first pass I knew we'd need to move beyond just the `jq` comparison, but I didn't have the confidence to start launching into SQL-centric content on my own, so happy to include this!